e2e-tests/functions

```diff
@@ -82,7 +94,7 @@ delete_pmm_api_key() {
 	fi
 
 	local ADMIN_PASSWORD
-	ADMIN_PASSWORD=$(kubectl -n "${NAMESPACE}" exec monitoring-0 -- bash -c "printenv | grep ADMIN_PASSWORD | cut -d '=' -f2")
+	ADMIN_PASSWORD=$(kubectl -n "${OPERATOR_NS:-$NAMESPACE}" exec monitoring-0 -- bash -c "printenv | grep ADMIN_PASSWORD | cut -d '=' -f2")
```

Comment:
I don't think we deploy PMM in operator namespace so this is probably wrong.

Reply:
[1eb58d9](https://github.com/percona/percona-server-mysql-operator/pull/673/commits/1eb58d9bed545ed7f16146e9f74e7f038c08100e)